documentation/modules/deploying/proc-deploy-kafka-connect-using-kafka-connect-build.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// assembly-using-kafka-connect-with-plugins.adoc
+
+[id='creating-new-image-using-kafka-connect-build-{context}']
+= Creating new container image automatically using Strimzi
+
+This procedure shows how to have Strimzi automatically build a new container image with additional connectors.
```

Comment:
```suggestion
This procedure shows how to configure Kafka Connect so that Strimzi automatically builds a new container image with additional connectors.
```